### DISCUSSION TITLE
Anyone is able to get changelog_from_git_commits to work via Github actions?

### DISCUSSION THREAD

**Discussion by KeiroMidori** [fastlane/fastlane#17784] [Q&A] [answered]
Basically this lane works fine on my machine. But as soon as I try it via github actions which runs the exact same machine than mine, the result will always be empty. 

Is anyone experiencing the same thing? What are your methods to get a changelog otherwise? 

EDIT: Never-mind I found out why. I needed to specify `fetch-depth: 0` to the checkout on GH Actions

**Comment by KeiroMidori** [comment, marked as answer]
Answered my own question --> see post edit

**Reply by Vaspory** [reply]
Great job!

**Comment by techinpark** [comment]
Thanks for share your solution. 👍 
https://github.com/actions/checkout
